advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Commvault enables your business to streamline management of its continuously evolving data environment, whether the data is on-premises or in the cloud.
+
+Commvault PostgreSQL iDataAgent provides the flexibility to backup PostgreSQL, EDB Postgres Advanced Server and EDB Postgres Extended Server databases in different modes and restore them when needed. You can perform a full/log backup or restore of database servers/individual databases/archive logs at any time and have full control over the process.
+
+Managing your data means knowing that it is protected and being able to effectively report on success or failure.  Through an easy-to-use interface, you can quickly check on the progress of your jobs to ensure things are moving as expected.  You can also use pre-built reports in an on-demand fashion, or scheduled, to keep you in the know.
+
+<p align="center">
+  <img src="Images/CommvaultEPASArchitecture.png">
```

Comment:
Need to update image.  
using our DB server image and add Extended server.
https://drive.google.com/drive/folders/1IjrxALM2N0fssOulY2r2EnA0TzO9KLE1
See if the png file I created looks good.  If not you can use the source material in folder to create a new one.